app/models/classroom.rb

```diff
@@ -0,0 +1,18 @@
+class Classroom < ActiveRecord::Base
+  validates :name, presence: true, uniqueness: true, length: { maximum: 255, too_long: "%{count} characters maximum" }
+  has_many :drills, dependent :destroy
+  has_many :drill_types, dependent :destroy
```

Comment:
I wanted some way to represent the functionality where the frequency of a drill type is specific to a classroom.  One of the classes only has to perform fire drills once per quarter while the rest of the classes must have these drills monthly.